classifiers/query_check/communication_style_classifier/__init__.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+import requests
+
+INPUT_EXAMPLE = {
+    "text": "Change the number in row 2 and 3.",
+    "model_name": "KernAI/multilingual-e5-communication-style",
+}
+
+
+class CommunicationStyleClassifierModel(BaseModel):
+    text: str
```

Comment:
Could also use `query` here instead of `text`. Makes the name more meaningful

Reply:
I would like to stick with `text`, as that is the default attribute we use for the refinery code and having it like this makes is a bit more similar.

Reply:
works for me :)